charts/opentelemetry-collector/values.yaml

```diff
@@ -192,9 +192,12 @@ containerLogs:
   enabled: false
 
 resources:
+  requests:
+    cpu: 100m
+    memory: 128Mi
```

Comment:
```suggestion
```
I think we still want it to requests its limits by default.  As the potential monitor of the system it would be bad if the collector got scheduled with insufficient resources and then couldn't scale up.

Reply:
makes sense, done!